src/NServiceBus.Persistence.CosmosDB.AzureStorageSagaExporter/NServiceBus.Persistence.CosmosDB.AzureStorageSagaExporter.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <PackageId>NServiceBus.Persistence.CosmosDB.AzureStorageSagaExporter</PackageId>
+    <PackageId>NServiceBus.Export.AspSagas</PackageId>
```

Comment:
How about

> Particular.Asp.ExportSagas

or

> Particular.Asp.Export

maybe the saga part is not that relevant

In short Particular is shorter than NServiceBus anyway :D

Reply:
Raised https://github.com/Particular/NServiceBus.Persistence.CosmosDB/pull/57

And we are back to Particular in the prefix :D